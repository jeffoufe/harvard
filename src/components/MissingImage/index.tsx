import React from 'react';
import useStyles from './styles';
import BrokenImageIcon from '@material-ui/icons/BrokenImage';

const MissingImage = () => {
    const classes = useStyles();
    return (
        <div className={classes.missingImageContainer}>
            <BrokenImageIcon className={classes.missingImageIcon} fontSize='large' />
        </div>
    )
}

MissingImage.displayName = 'MissingImage';
export default MissingImage;