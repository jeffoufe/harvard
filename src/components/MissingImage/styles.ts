import { makeStyles, createStyles } from '@material-ui/core/styles';

export default makeStyles(() =>
    createStyles({
        missingImageContainer: {
            position: 'relative',
            width: '100%',
            height: 200,
        },
        missingImageIcon: {
            position: 'absolute',
            top: '50%',
            left: '50%',
            transform: 'translate(-50%, -50%)'
        }
    })
)