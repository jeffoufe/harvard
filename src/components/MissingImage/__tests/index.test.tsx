import React from 'react';
import { shallow } from 'enzyme';
import MissingImage from '..'; 

describe('MissingImage', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<MissingImage />);
        expect(wrapper).toMatchSnapshot();
    })
})