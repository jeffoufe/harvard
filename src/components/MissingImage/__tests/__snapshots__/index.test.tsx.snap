// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissingImage renders correctly 1`] = `
<div
  className="makeStyles-missingImageContainer-1"
>
  <BrokenImageIcon
    className="makeStyles-missingImageIcon-2"
    fontSize="large"
  />
</div>
`;
